src/App.js

```diff
@@ -37,7 +37,9 @@ const tweetsArray = [
 function App() {
   return (
     <div className="App">
-      <Tweet />
```

Comment:
what if we have 5000 objects in that array this is not dinamic just make a map from 

```
tweetsArray.map( item => ...)

```